### PR TITLE
Automatic custom fonts registration for iOS.

### PR DESCRIPTION
**Pull request by nsndeck**

### CODE REVIEW

**Comment by nsndeck** [comment]
run ci

**Comment by nsndeck** [comment]
run ci

**Comment by dtopuzov** [comment]
:+1:

File: ui/styling/font.ios.ts [hunk 1]
```diff
@@ -159,7 +159,7 @@ export module ios {
 
         var error = new interop.Reference();
         if (!CTFontManagerRegisterGraphicsFont(font, error)) {
-            throw new Error(CFErrorCopyDescription(<NSError>error.value));
+            console.log("Error occur while registering font: " + CFErrorCopyDescription(<NSError>error.value));
```
Comment (vakrilov) [inline ui/styling/font.ios.ts]: Better use:
```
trace.write( ... , trace.categories.Error, trace.messageType.error);
```

**Pull request merged**